docs/api/_index.md

```diff
@@ -109,6 +110,14 @@ GET /config
 
 Displays the configuration currently applied to Cortex (in YAML format), including default values and settings via CLI flags. Sensitive data is masked. Please be aware that the exported configuration **doesn't include the per-tenant overrides**.
 
+### Runtime Configuration
+
+```
+GET /runtime_config
+```
+
+Displays the runtime configuration currently applied to Cortex (in YAML format), including default values.
```

Comment:
Please add a note saying that if Cortex is not configured with any `-runtime-config.file`, then this will not be available.